gtsfm/data_association/data_assoc.py

```diff
@@ -162,6 +179,18 @@ def run(
             ],
         )
 
+        if gt_computed_exit_codes_distribution is not None:
+            for (gt_exit_code, computed_exit_code), count in gt_computed_exit_codes_distribution.items():
+                # We have assigned two tags to each track: the triangulation exit codes w.r.t ground truth cameras
```

Comment:
It would be less confusing if you called the thing by the same name: tag, classification, exit code. I'd suggest exit code since its already used. So this sentence can be rephrased as: 
Each track is assigned two exit codes (one with respect to GT cameras, other wrt computed cameras).

Reply:
better?

Reply:
updated once more